tests/local-suite/packaging.tests.js

```diff
@@ -0,0 +1,122 @@
+    const templateName = 'aws-nodejs';
+    execSync(`${serverlessExec} create --template ${templateName}`, { cwd });
+    execSync(`${serverlessExec} package`, { cwd });
+    return testUtils.listZipFiles(path.join(cwd, '.serverless/aws-nodejs.zip'))
+      .then(zipfiles => {
+        expect(zipfiles).toEqual(['handler.js']);
+      });
+  });
+
+  it('packages the default aws template with an npm dep correctly', () => {
+    const templateName = 'aws-nodejs';
+    execSync(`${serverlessExec} create --template ${templateName}`, { cwd });
+    execSync('npm init --yes', { cwd });
+    execSync('npm i lodash', { cwd });
+    execSync(`${serverlessExec} package`, { cwd });
+    return testUtils.listZipFiles(path.join(cwd, '.serverless/aws-nodejs.zip'))
+      .then(zipfiles => {
+        const nodeModules = new Set(
+          zipfiles.filter(f => f.startsWith('node_modules')).map(f => f.split(path.sep)[1]));
+        const nonNodeModulesFiles = zipfiles.filter(f => !f.startsWith('node_modules'));
+        expect(nodeModules).toEqual(new Set(['lodash']));
+        expect(nonNodeModulesFiles).toEqual(['handler.js', 'package-lock.json', 'package.json']);
+      });
+  });
+
+  it('doesn\'t package a dev dependency', () => {
+    const templateName = 'aws-nodejs';
+    execSync(`${serverlessExec} create --template ${templateName}`, { cwd });
+    execSync('npm init --yes', { cwd });
+    execSync('npm i --save-dev lodash', { cwd });
+    execSync(`${serverlessExec} package`, { cwd });
+    return testUtils.listZipFiles(path.join(cwd, '.serverless/aws-nodejs.zip'))
+      .then(zipfiles => {
+        const nodeModules = new Set(
+          zipfiles.filter(f => f.startsWith('node_modules')).map(f => f.split(path.sep)[1]));
+        const nonNodeModulesFiles = zipfiles.filter(f => !f.startsWith('node_modules'));
+        expect(nodeModules).toEqual(new Set([]));
+        expect(nonNodeModulesFiles).toEqual(['handler.js', 'package-lock.json', 'package.json']);
+      });
+  });
+
+  it('ignores package json files per ignore directive', () => {
+    const templateName = 'aws-nodejs';
+    execSync(`${serverlessExec} create --template ${templateName}`, { cwd });
+    execSync('npm init --yes', { cwd });
+    execSync('echo \'package: {exclude: ["package*.json"]}\' >> serverless.yml', { cwd });
+    execSync('npm i lodash', { cwd });
+    execSync(`${serverlessExec} package`, { cwd });
+    return testUtils.listZipFiles(path.join(cwd, '.serverless/aws-nodejs.zip'))
+      .then(zipfiles => {
+        const nodeModules = new Set(
+          zipfiles.filter(f => f.startsWith('node_modules')).map(f => f.split(path.sep)[1]));
+        const nonNodeModulesFiles = zipfiles.filter(f => !f.startsWith('node_modules'));
+        expect(nodeModules).toEqual(new Set(['lodash']));
+        expect(nonNodeModulesFiles).toEqual(['handler.js']);
+      });
+  });
+});
+
+describe('cloudformation packaging', () => {
```

Comment:
Can this also be wrapped in a "global" `Integration Test - Packaging` `describe` block?